components/organisms/o-search-panel.vue

```diff
@@ -239,10 +239,15 @@ export default {
   }
 
   .no-results {
-    height: 5rem;
-    line-height: 5rem;
     display: flex;
     justify-content: center;
+    height: 5rem;
+    line-height: var(--spacer-base);
```

Comment:
Line height should not be set using spacer. Spacers should only used for spacing related use cases like: gap, padding and margin.

Reply:
For this issue I think just changing font size is enough. don't need anything else @azkyakhan

Reply:
Ok done sir